src/Pelagos/Bundle/AppBundle/Resources/public/js/dataset-submission.js

```diff
@@ -727,6 +727,18 @@ $(function() {
                 source.append(option);
                 source.append(sortOptions(source.find("option").detach()));
             }
+        } else if ($(event.currentTarget).text() == "up") {
```

Comment:
Could we assign an id to the button element and use the identifier, instead of the value?